packages/ember-runtime-config/package.json

```diff
@@ -121,7 +124,7 @@
     "./middleware": "./middleware/index.js"
   },
   "typesVersions": {
-    "*": {
+    ">=5.0.0": {
```

Comment:
@vstefanovic97 why do we need this?

Reply:
We use `allowImportingTsExtensions` option which is only supported by version 5 or higher